eth_tester/backends/pyevm/main.py

```diff
@@ -490,6 +497,17 @@ def estimate_gas(self, transaction):
 
         return self.chain.estimate_gas(spoofed_transaction)
 
+    def is_eip838_error(self, error):
+        if not isinstance(error, EVMRevert):
+            return False
+        elif len(error.args) == 0:
+            return False
+
+        try:
+            return error.args[0][:4] == EIP838_SIG
```

Comment:
I suspect this can also throw an `IndexError` were the exception to have no arguments.  Probably worth handling.